test-engineering/load-tests/load_test_setup.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+set -eu
+
+#Setup environment
```

Comment:
The lines from here up to line 29 seem to be about declaring variables, not about setting up the environment.

Reply:
```suggestion
#Declare variables
```